.travis.yml

```diff
@@ -2,13 +2,17 @@
 # based on http://anil.recoil.org/2013/09/30/travis-and-ocaml.html
 
 language: c
+sudo: required
 
-script:
-  - bash -ex .travis-ci.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+
+script: bash -ex .travis-opam.sh
 
 env:
-  - OCAML_VERSION=4.01.0 OPAM_VERSION=1.2.0
-  - OCAML_VERSION=4.02.1 OPAM_VERSION=1.2.0
+  - OCAML_VERSION=latest
```

Comment:
Might want to pin one or two versions (like 4.06) instead of using `latest`.

Reply:
doesn't work, no time to look at it. So Lastest it will be until someone changes it